setup.py

```diff
@@ -8,14 +8,14 @@
 console_scripts = ['dexbot-cli = dexbot.cli:main']
 install_requires = [
     'bitshares==0.2.1',
-    'uptick==0.2.0',
+    'uptick>=0.2.0',
     'click',
     'sqlalchemy',
     'ruamel.yaml>=0.15.37',
     'sdnotify',
     'appdirs>=1.4.3',
-    'pycryptodomex==3.6.4',
-    'websocket-client==0.54.0'
+    'pycryptodomex>=3.6.4',
+    'websocket-client>=0.53.0'
```

Comment:
Why downgrade websocket-client? 0.53.0 has issues with python-bitshares.

Reply:
Because when I build with pyqt5==5.11.3 on OSX, I get this error, and only by downgrading these two items does it build on OSX. Your suggestions? @PermieBTS 

Traceback (most recent call last):
  File "/Users/christina/Documents/GitHub/dexenv-test/bin/pyrcc5", line 6, in <module>
    from PyQt5.pyrcc_main import main
  File "/Users/christina/Library/Python/3.7/lib/python/site-packages/PyQt5/pyrcc_main.py", line 21, in <module>
    from PyQt5.QtCore import PYQT_VERSION_STR, QDir, QFile
ModuleNotFoundError: No module named 'PyQt5.sip'
Traceback (most recent call last):
  File "/Users/christina/Library/Python/3.7/lib/python/site-packages/pyqt_distutils/build_ui.py", line 95, in run
    subprocess.check_call(cmd)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 328, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['pyrcc5', '/Users/christina/Documents/GitHub/DEXBot/dexbot/resources/fonts.qrc', '-o', '/Users/christina/Documents/GitHub/DEXBot/dexbot/resources/fonts_rc.py']' returned non-zero exit status 1.

Reply:
> 0.53.0 has issues with python-bitshares

Would refactoring pybitshares in the process of async development be able to solve this issue?

Reply:
@PermieBTS there is no issues currently, it was already fixed.
@thehapax I asked about websocket-client not pyqt5.

Reply:
don't need upgrade to websocket-client. only need downgrade on pyqt5  for os X